native/src/main/java/io/ballerina/stdlib/graphql/runtime/client/QueryExecutor.java

```diff
@@ -33,17 +31,17 @@
 public class QueryExecutor {
 
     /**
-    * Executes the GraphQL document when the corresponding Ballerina remote operation is invoked.
-    */
+     * Executes the GraphQL document when the corresponding Ballerina remote operation is invoked.
+     */
     public static Object execute(Environment env, BObject client, BString document, Object variables,
                                  Object operationName, Object headers, BTypedesc targetType) {
         return invokeClientMethod(env, client, document, variables, operationName, headers, targetType,
                 "processExecute");
     }
 
     /**
-    * Executes the GraphQL document when the corresponding Ballerina remote operation is invoked.
-    */
+     * Executes the GraphQL document when the corresponding Ballerina remote operation is invoked.
+     */
```

Comment:
Maybe the IDE formats these incorrectly?

Reply:
This is formatted by the IDE. 
But this formatting is correct we are following the same in other places.

Reply:
My bad.